src/main/java/com/mercuryCyclists/Inventory/service/ProductService.java

```diff
@@ -93,15 +93,27 @@ public boolean addPart(Long productId, Part part) {
         return true;
     }
 
+    /**
+     * Update part helper method
+     * @param product
+     * @param part
+     * @return If given product contains given part return true else false
+     */
+    private boolean updatePartHelper(Product product, Part part) {
+        return product.getSet().contains(part);
+    }
+
```

Comment:
My bad, I now get why this is for.
UpdatePartHelper is not the best naming but I'm happy with it